.github/analyse.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-set -e
-
-function finish() {
-  make logs stop
```

Comment:
logs won't be printed anymore? Not a big loss

Reply:
Probably useful to keep for when it fails.